integration-tests/helpers.js

```diff
@@ -172,11 +172,19 @@ function spawnProc (filename, options = {}, stdioHandler) {
         }
         resolve()
       })
-    if (stdioHandler) {
-      proc.stdout.on('data', (data) => {
+
+    proc.stdout.on('data', data => {
+      if (stdioHandler) {
         stdioHandler(data)
-      })
-    }
+      }
+      // eslint-disable-next-line no-console
+      console.log(data.toString())
+    })
+
+    proc.stderr.on('data', data => {
+      // eslint-disable-next-line no-console
+      console.log(data.toString())
```

Comment:
let's make this `console.error`, so that stderr is always sent to stderr and stdout to stdout